nss-day-cohort-44/rare-rest-api-malicous-vortices
# Kn create post server side

# ISSUE TICKET 
## Description
synced up the data required on the server side with data sent by the client side when creating a new post.


## Type of change
- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions
`git fetch origin Kn-create-post-server-side`
`git checkout Kn-create-post-server-side` 

User Instructions: 
Pull this down along with the Kn-create-post pull request on the client side run `cd` into both of them and run them
once that is done, go to the client and click add new post, create a test post and you should see it on the list where you were redirected.

# Checklist:
